pyaedt/generic/configurations.py

```diff
@@ -984,6 +995,48 @@ def _update_datasets(self, data_dict):
                 data_dict["vunit"],
             )
 
+    @pyaedt_function_handler()
+    def validate(self, config):
+        """Validate a configuration file against the schema. The default schema
+            can be found in ``pyaedt/misc/config.schema.json``.
+
+        Parameters
+        ----------
+        config : str, dict
+            Configuration as json (file) or dict.
+
+        Returns
+        -------
+        bool
+            ``True`` if the configuration file is valid, otherwise ``False``.
+            If the validation fails, a warning will also be written to the logger.
```

Comment:
```suggestion
            ``True`` if the configuration file is valid, ``False`` otherwise.
            If the validation fails, a warning is also written to the logger.
```
Use present tense.